src/indicator/algorithm/calculator/undefinedInputAdapter.js

```diff
@@ -1,54 +0,0 @@
-import d3 from 'd3';
-import slidingWindow from './slidingWindow';
-
-// Indicator algorithms are not designed to accomodate leading 'undefined' value.
-// This adapter adds that functionality by adding a corresponding number
-// of 'undefined' values to the output.
-export default function() {
```

Comment:
Previously used by envelope algorithm, with extra null check added in envelope renderer to compensate removal. Should this stay / move into d3fc-technical-indicator?